ghc-lib-gen/src/GhclibgenFlavor.hs

```diff
@@ -77,6 +79,7 @@ ghcSeries = \case
   f | Ghc941 <= f && f < Ghc961 -> GHC_9_4
   f | Ghc961 <= f && f < Ghc981 -> GHC_9_6
   f | Ghc981 <= f && f < Ghc9101 -> GHC_9_8
-  f | Ghc9101 <= f && f < GhcMaster -> GHC_9_10
-  GhcMaster -> GHC_9_12
+  f | Ghc9101 <= f && f < Ghc9121 -> GHC_9_10
+  f | Ghc9121 <= f && f < GhcMaster -> GHC_9_14
```

Comment:
Did you mean to write `GHC_9_12` here? I would assume so judging by the pattern?